.changeset/stale-donkeys-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/adapter-node": minor
+---
+
+feat: add graceful shutdown
```

Comment:
```suggestion
"@sveltejs/adapter-node": major
---

breaking: add graceful shutdown
```

I _think_ we should call this a breaking change. I'm not 100% certain what sorts of scenarios would break, but we are changing existing behavior, not just adding new behavior.